Makefile

```diff
@@ -61,8 +61,7 @@ test: run-unit-tests
 
 .PHONY: lint
 lint: build-slim build-test
-	# Note: Make sure that you run `git config diff.noprefix false` in this repo
-	# See this issue for more details: https://github.com/golangci/golangci-lint/issues/948
+	if [ $$(git config --get diff.noprefix) == "true" ]; then echo -e "\nLinting won't work because you have a git setting 'diff.noprefix' set to 'true',\nwhich does not work reliably with the golangci-lint --new-from-rev flag. \n\nTo run the linter, set 'diff.noprefix' setting to 'true' for this repository using the following command:\n\n'git config diff.noprefix false'\n\nSee https://github.com/golangci/golangci-lint/issues/948 for more details."; exit 1; fi
```

Comment:
`echo -e` is not portable (doesn't work on dash, for example). Please use `printf`

Reply:
As said, I think it will never use dash :)

Reply:
https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html

Reply:
What you said is actually the opposite ;)

Also, we really can't know which shell the user is using, using printf is a trivial change, `echo -e` is not recommended since years... I really don't see a point arguing about this.